qiskit/wrapper/configrc/_configrc.py

```diff
@@ -0,0 +1,268 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""
+A module for generating and manipulating the qiskitrc file.
+"""
+import os
+import ast
+import configparser
+from qiskit import QISKitError
+from qiskit.wrapper.configrc import _settings as rc_set
```

Comment:
QISKit already uses the following:
```python
import numpy as np
import scipy.linalg as la
import scipy.sparse as sp
import scipy.sparse.csgraph as cs
import networkx as nx
from sympy import Number as N
from qiskit.qasm import _node as node
import ply.lex as lex
from . import _node as node
import ply.yacc as yacc
import matplotlib.pyplot as plt
```
why is my usage of the same functionality problematic?

Reply:
Again, for readability - in general they do not provide tangible benefits, but have the drawback add another level of indirection in the code, which hinders parsing the code at a glance. The examples you list are either references to other libraries (where some of them, as the numpy alias, are relatively widespread) with only three of them related to `qiskit` modules (and part of code that has not changed much since the beginning of the project).